WORKSPACE

```diff
@@ -52,7 +52,7 @@ load("@bazel_skylib//lib:unittest.bzl", "register_unittest_toolchains")
 register_unittest_toolchains()
 
 # gazelle:repository go_repository name=co_honnef_go_tools importpath=honnef.co/go/tools
-# gazelle:repository go_repository name=com_github_bazelbuild_buildtools importpath=github.com/bazelbuild/buildtools build_naming_convention=go_default_library
+# gazelle:repository go_repository name=com_github_bazelbuild_buildtools importpath=github.com/bazelbuild/buildtools
```

Comment:
Did you update this to the latest version including the alias change? If it doesn't work, just remove this update and revert this line and we handle it separately.

Reply:
The BCR test explicitly downgraded buildtools. This is why the test was red.